src/models/api-key.ts

```diff
@@ -181,15 +190,21 @@ const getApiKeysModel = function (
 		 */
 		async getProvisioningApiKeysByApplication(
 			slugOrUuidOrId: string | number,
-			options: BalenaSdk.PineOptions<BalenaSdk.ApiKey> = {},
-		): Promise<BalenaSdk.ApiKey[]> {
+			options: ODataOptionsWithoutCount<ApiKey['Read']> = {},
+		): Promise<Array<ApiKey['Read']>> {
 			const appOpts = {
-				$select: 'actor' as const,
-			};
+				$select: 'actor',
+			} as const;
```

Comment:
Double-checking: is moving the `as const` to the whole opts instead of just the `$select` intentional?

Reply:
Yes, this is a pattern we can use everywhere and simplify the thinking of where we should put stuff. It is also what we do on the api, so we just `as const` the whole options object everytime and things should just work